### DISCUSSION TITLE
How to add new langauge in pimcore

### DISCUSSION THREAD

**Discussion by abdelrhmanyosry** [pimcore/pimcore#16819] [Q&A / Support]
how to add a new language in pimcore & how can i extend the admin panel in pimcore 
![image](https://github.com/pimcore/pimcore/assets/121692656/0553bade-1a5a-4f7f-84e0-c34050cc51c4)

**Comment by skoch98** [comment]
The question will occurre just more questions 🫣

What do you mean exactly?

If you need content languages:
Like the screenshot said: Add the content languages you want to have and save it.
If the save button is disabled you need to enable the APP_DEBUG env variable or choose the system settings as write-target. See https://pimcore.com/docs/platform/Pimcore/Deployment/Configuration_Environments/#page_Configuration-Storage-Locations-Fallbacks

If you want to add a new admin language (for the admin ui texts) take a look at the docs: https://pimcore.com/docs/platform/Pimcore/Multi_Language_i18n/Admin_Translations/#adding-your-own-admin-languages-since-v636

^Sebastian
